src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const DEFAULT_DEMAGE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.name = name;
+    this.health = DEFAULT_HEALTH;
+    Animal.alive.push(this);
+  }
+
+  demage(amout = DEFAULT_DEMAGE) {
+    this.health -= amout;
+
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((item) => item !== this);
+    }
+  }
```

Comment:
1) Move this method to `Carnivore` class, cuz u `Herbivore` should not have access to this method
2) `item !== this`? U need to remove all values ​​that are equal to 0 or less. Change the condition in ur filter
3) After the move, u won't need to create a (damage) method to perform the action)